index.bs

```diff
@@ -3676,85 +3690,85 @@ the same procedure as other [=assertion signatures=] generated by the [=authenti
             <tr>
-            <th>Bit 3 value</th>
-            <th>Bit 4 value</th>
+            <th>`BE`</th>
+            <th>`BS`</th>
             <th>Description</th>
         </tr>
         <tr>
             <td>`0`</td>
             <td>`0`</td>
             <td>
-                The credential is a single-device credential and cannot become a multi-device credential
+                The credential is a [=single-device credential=] and cannot become a [=multi-device credential=].
             </td>
         </tr>
         <tr>
             <td>`0`</td>
             <td>`1`</td>
             <td>
-                This combination is not allowed
+                This combination is not allowed.
             </td>
         </tr>
         <tr>
             <td>`1`</td>
             <td>`0`</td>
             <td>
-                The credential is currently a single-device credential but may become a multi-device credential in the future
+                The credential is currently a [=single-device credential=] but may become a [=multi-device credential=] in the future.
             </td>
         </tr>
         <tr>
             <td>`1`</td>
             <td>`1`</td>
             <td>
-                The credential is a multi-device credential
+                The credential is a [=multi-device credential=].
             </td>
         </tr>
     </table>
     <figcaption>
-        [=Authenticator data=] bits 3 and 4 combinations 
+        `BE` and `BS` [=flag=] combinations
     </figcaption>
 </figure>
 
-It is recommmended that [=Relying Party|Relying Parties=] store the last value of these bits with the user account for future evaluation.
+It is RECOMMENDED that [=[RPS]=] store the most recent value of these [=flags=] with the [=user account=] for future evaluation.
 
-The following is a non-normative, non-exhaustive list of how [=Relying Party|Relying Parties=] may utilize these bits:
+The following is a non-normative, non-exhaustive list of how [=[RPS]=] might utilize these [=flags=]:
 
     - Requiring additional [=authenticators=]:
 
-        When bit 3 is set to `0`, the [=authenticator=] will never allow the credential to transition from a single-device credential to
-        a multi-device credential. 
+        When `BE` [=flag=] is set to `0`, the [=generating authenticator=] will never allow the credential to transition from a 
+        [=single-device credential=] to a [=multi-device credential=]. 
         
-        A single-device credential is not durable and cannot survive single device loss. [=Relying Party|Relying Parties=]
-        should ensure that an account has additional [=authenticators=] enrolled and/or an account recovery process in place.
+        A [=single-device credential=] is not durable and cannot survive single device loss. [=[RPS]=] should ensure that a [=user account=] 
+        has additional [=authenticators=] [=registration ceremony|registered=] and/or an account recovery process in place.
```

Comment:
We haven't defined what "durable" is supposed to mean in this context, and I've seen the term frowned upon as it implies that non-syncing authenticators are "not durable". I think it's better to slightly rephrase this.

```suggestion
        A [=single-device credential=] is not resilient to single device loss. [=[RPS]=] should ensure that a [=user account=] 
        has additional [=authenticators=] [=registration ceremony|registered=] and/or an account recovery process in place.
```